### PR TITLE
Setup Telemetry Manager Repository

### PR DESCRIPTION
**Pull request by hisarbalik**
**Description**

Changes proposed in this pull request:

- Code owner list updated
- Dummy code and dummy tests added
- Readme, license, contribution and code of conducts added

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/16745

### CODE REVIEW

File: common/makefiles/generic-make-go.mk [hunk 1]
```diff
@@ -0,0 +1,249 @@
+# Default configuration
```
Comment (chrkl) [inline common/makefiles/generic-make-go.mk:1]: Let's get rid of this immediately. As far as I could see in test-infra, we only use the `lint` and `test-local` targets from our own Makefile. Let's revert this to something close to a "normal" Kubebuilder project Makefile.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+##@ Development
+
+# Generate manifests e.g. CRD, RBAC etc.
+manifests-local: controller-gen-local
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+
```
Comment (skhalash) [inline Makefile:47]: Would be nice if we could get rid of the local suffixes

**Review by skhalash — CHANGES_REQUESTED** [review]

**Review by skhalash — APPROVED** [review]

**Pull request merged**